xapian-applications/omega/templates/query

```diff
@@ -52,7 +52,7 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 </script>
 </head>
 <body>
-<form name=P method=get
+<form id="query-form" name=P method=get
```

Comment:
I'd probably just use `id=query`. The CSS selector `form#query` reads better to me than `form#query-form` :-)